src/components/HeaderWithBackButton.js

```diff
@@ -87,15 +78,12 @@ const defaultProps = {
     title: '',
     subtitle: '',
     onDownloadButtonPress: () => {},
-    onCloseButtonPress: () => {},
-    onBackButtonPress: () => {},
+    onBackButtonPress: Navigation.goBack,
     onThreeDotsButtonPress: () => {},
-    shouldShowBackButton: false,
```

Comment:
Was there any place where the `shouldShowBackButton` was false? are we covering that case?

Reply:
There we go